app/views/deposits/recurring/edit_account_application.html

```diff
@@ -7,7 +7,7 @@
     </ul>
 </div>
 <api-validate></api-validate>
-<form name="editrecurringdepositaccountform" novalidate="" class="well form-inline" rc-submit="submit()">
+<form name="editrecurringdepositaccountform" novalidate="" class="well form-inline card" rc-submit="submit()">
```

Comment:
Can you provide a screenshot of this. I think this will not work. 
If works - then fine, otherwise enclose <form> in div class 'card' similar to edit page of FD